util/dnsname/dnsname.go

```diff
@@ -20,14 +20,12 @@ const (
 // A FQDN is a fully-qualified DNS name or name suffix.
 type FQDN string
 
-func ToFQDN(s string) (FQDN, error) {
+// Retains leading '.' on FQDN to populate Hosts with wildcard entries from ExtraRecords
```

Comment:
See https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences for Go doc style. It should be a complete sentence starting with the name of the func and ending in a period.

But this is a util package that isn't about tailcfg or even Tailscale so it can't mention things like ExtraRecords which is a very Tailscale wire protocol concept.